test/msw-api/setup-server/scenarios/browser.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from 'path'
+import { TestAPI, runBrowserWith } from '../../../support/runBrowserWith'
+import { captureConsole } from '../../../support/captureConsole'
+
+describe('API: setupWorker / start', () => {
+  let test: TestAPI
+
+  beforeAll(async () => {
+    test = await runBrowserWith(path.resolve(__dirname, 'browser.mocks.ts'))
+  })
+
+  afterAll(() => {
+    return test.cleanup()
+  })
+
+  it('should throw an error and print to console', async () => {
+    const errors: string[] = []
+
+    captureConsole(test.page, errors, (message) => {
+      return message.type() === 'error'
+    })
+
+    await test.reload()
+
+    const nodeMessage = errors.find((message) => {
+      return message.includes(
+        '[MSW] It appears you are trying to set up Node Integration',
```

Comment:
Could you please also change this to reflect the updated error message?

```js
message.startsWith('[MSW] Failed to execute `setupServer` in the environment that is not NodeJS')
```

Thanks!